comps/navbar.js

```diff
@@ -14,15 +31,15 @@ function Navbar(props){
 	        		</a>
 	        	</Link>
 
-	        	<div className="inline-flex h-full w-5/12 justify-end float-right">
+	        	<div className="inline-flex h-full w-5/12 justify-end float-right mt-3">
 	        		
-	        		<div className="inline-flex bg-gradient-to-r from-UMOgreen via-UMOblue to-UMOpurple mr-5 mt-2 h-2 w-8/12 rounded-md">
-	        			<div className="w-5/12 rounded-l-md h-2">
+	        		<div className="inline-flex justify-end bg-gradient-to-r from-UMOgreen via-UMOblue to-UMOpurple mr-5 h-2 w-8/12 rounded-md mt-2">
+	        			<div className={"w-" +(full ? "0" : calc) +" bg-light h-full clipwacsd rounded-r-md transition-width ease-in duration-300"}>
```

Comment:
watch out here. This class might be removed by the PostCSS optimizer (the `w-0` or `w-XX` ones)